experiments/azure/cyclecloud/cpu/experiment/configs/single-node-benchmarks/slurm-single-node-benchmarks.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+sudo /shared/bin/singularity exec /shared/containers/metric-single-node_cpu-zen4-tmpfile.sif \
+    /bin/bash /entrypoint.sh > ../data/single-node-benchmarks/$( hostname )-single-node.out
```

Comment:
Do we want `&>` here to capture error too?

Reply:
Since it's not submitted via `sbatch`, yes, we do want that. Good catch.